### PR TITLE
Shift Rows and Inv Shit Rows added aswell as testing unit for both me…

### PR DESCRIPTION
**Pull request by osabga**
…thods.

Shift Rows and Inv Shit Rows added aswell as testing unit for both methods.

### CODE REVIEW

File: Transformations/shiftRowsUnitTesting.cpp [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+                     << "AT: " << i << "," << j << endl;
+                return false;
+            }
+        }
+    }
+
+    return true;
+}
+
+bool invShiftRowsTest1()
```
Comment (samsbg) [inline Transformations/shiftRowsUnitTesting.cpp:109]: No sería 4?

Comment (osabga) [reply Transformations/shiftRowsUnitTesting.cpp:109]: Como que 4?

Comment (samsbg) [reply Transformations/shiftRowsUnitTesting.cpp:109]: My bad, see now

**Review by samsbg — APPROVED** [review]
Try to finish mix columns as soon as possible now that they're in the main, but looks good to me

**Pull request merged**